app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/deployManifestConfig.html

```diff
@@ -44,4 +44,12 @@ <h4>Manifest Configuration</h4>
                               accounts="ctrl.metadata.backingData.artifactAccounts"
                               expected-artifacts="ctrl.expectedArtifacts">
   </expected-artifact-selector>
+
+  <hr>
+
+  <expected-artifact-multi-selector command="ctrl.$scope.stage"
+                                    ids-field="'deployArtifactIds'"
```

Comment:
If you're using fixed values on attributes, more idiomatic to use `'@'` instead of `'<'` on the bindings definition. That means that you just pick up the attribute's literal value, and you don't have to wrap the value in quotes:
```
ids-field="deployArtifactIds"
artifact-label="Artifacts to Deploy"
```

Reply:
Oh neat